### ISSUE TITLE
Pushing current playing media to selected source

### ISSUE THREAD

**Issue by xMuchux** [kalkih/mini-media-player#610]
Hi all,

Fairly new to HASS.IO so forgive me if my question has a simple answer.

I've managed to get the mini-media-player working and grouped some players, however 2 questions:

1. Is it possible to push the currently playing media to a different source via mini media player (please note: I have spotcast also configured, however, after searching around I have not been able to figure out if I can use to answer my question)
2. Why does my 2nd source (Dining Room ChromeCast) not show up in the sources dropdown?

![screenshot](https://user-images.githubusercontent.com/99677766/154728339-88af6d5f-726b-44be-98e4-1a5600d3a677.png)

My code in the card:

```
type: entities
entities:
  - type: custom:mini-media-player
    name: Spotify Mini Player
    entity: media_player.spotify_*USERNAME*
    group: true
    source: icon
    sound_mode: full
    power_state: true
    info: scroll
    artwork: cover-fit
  - type: custom:mini-media-player
    name: Dining Room Chromecast
    entity: media_player.dining_chromecast_audio
    group: true
    artwork: cover-fit
    source: icon
    sound_mode: icon
    controls: true
    info: scroll
state_color: false
```

Thank you in advance!

**Comment by kalkih** [comment]
Hey,

1. mini-media-player is built to handle one media player entity 1:1 relationship, so depending on the media player entity, this may or may not work, if it's an amplifier or another intergration with multiple zones/sources, it should work out of the box. As for the Spotify intergration and your case it would probably have worked as expected if Cast devices were discovered and picked up by the Spotify integration, see 2.
2. iirc the core Spotify integration doesn't pick up idle Cast devices, hence to popularity of Spotcast.

So to summarize; Not in this case unfortunately.

**Comment by xMuchux** [comment]
Alright, super. Thanks for the explanation of the 1:1 relationship and I observed the idle cast devices issue as well. I'll find a way around it.
